backend/frontend/static/frontend/user_board.js

```diff
@@ -21,3 +21,54 @@ function moveCardToCorrectColumn(cardId, column) {
     destinationColumn.insertBefore(card, destinationColumn.firstChild);
   }
 }
+
+function showCardAlert({
+  cardId,
+  message,
+  alertType = "info",
+  persist = true,
+}) {
+  const alertThemes = {
+    success: "border-green-300 bg-green-100 text-green-800",
+    error: "border-red-300 bg-red-100 text-red-800",
+    warning: "border-yellow-300 bg-yellow-100 text-yellow-800",
+    info: "border-blue-300 bg-blue-100 text-blue-800",
+  };
+
+  const iconThemes = {
+    success: "fa-check-circle",
+    error: "fa-times-circle",
+    warning: "fa-exclamation-circle",
+    info: "fa-info-circle",
+  };
+
+  const card = document.getElementById(`card_${cardId}`);
+  const alert = document.createElement("div");
+  const cardButtons = card.querySelectorAll("button");
+
+  cardButtons.forEach((cardButton) => (cardButton.disabled = true));
+
+  alert.className = `mb-3 relative leading-tight flex gap-2 rounded-xl border p-3 text-sm ${alertThemes[alertType]}`;
+
+  alert.innerHTML = `
+      <i class="fas ${iconThemes[alertType]} self-center"></i>
```

Comment:
This works but it is not ideal that the html is embedded in a js file.

Is there a way to do this with templates rather?

Eg have a template that draws the card and the alert, and also calls a function to hide the alert after 5 seconds? 

The template would look something like:

```
{% include "frontend/user/board/partial_alert.html" %}
{% include "frontend/user/board/partial_card.html" %}

<script>
  hideCardAlert()
</script>
```

Then we could reuse the alert in different places in the app if needed.

Reply:
Another option is to include the alert stuff inside partial_card.html and it only shows up if there is a message.